h/presenters/group_json.py

```diff
@@ -10,27 +10,42 @@ def __init__(self, group, links_svc=None):
         self.group = group
         self._links_svc = links_svc
 
-    def asdict(self):
-        return self._model(self.group)
+    def asdict(self, expand=[]):
+        model = self._model()
+        self._expand(model, expand)
+        self._inject_urls(model)
+        return model
+
+    def _expand(self, model, expand=[]):
+        if 'organizations' in expand:
```

Comment:
The field name is `organization` but the expected argument to `expand` here is `organizations`, which isn't documented. Depending on whether `expand` applies to a specific field or to all resources of a particular type, I think it would be good to have some consistent rule for determining the name of the `expand` parameter which can be documented once and apply everywhere that we use this.